.gitignore

```diff
@@ -76,3 +76,6 @@ dist
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+# i18next parser creating moduleName folder when parsing index.ts
+moduleName
```

Comment:
Might it be obvious to you, @ibacher, why this is happening?

Reply:
It's [this line](https://github.com/openmrs/openmrs-esm-home/blob/620836570958d5b775470e3b6f924b7947a36601/packages/esm-home-app/src/index.ts#L34) that is telling it to use `moduleName` as a namespace combined with [this rule](https://github.com/openmrs/openmrs-esm-home/blob/620836570958d5b775470e3b6f924b7947a36601/tools/i18next-parser.config.js#L52C6-L52C6) for what files are generated. Maybe we should just hard-code `translations/$LOCALE.json` for that value?